core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -272,17 +287,19 @@ private boolean updatePlugins( final WildcardMap<ProjectVersionRef> relocations,
                 dependency.setGroupId( pluginReference.groupIdNode.getTextContent() );
                 dependency.setArtifactId( pluginReference.artifactIdNode.getTextContent() );
 
-                final boolean isUpdate = relocations.containsKey( dependency );
+                final boolean isUpdate = dependencyRelocations.containsKey( dependency );
```

Comment:
Is `containsKey` necessary, or will it get return `null` just in this case? Then, we could combine this line and 294 as

```
final ProjectVersionRef relocation = dependencyRelocations.get( dependency );

if ( relocation != null )
```
```

Reply:
I've not changed this apart from renaming the variable so this your original code.

Reply:
But I do agree with your assessment and will change it.